src/main/java/org/myrobotlab/swing/abstracts/AbstractSpeechSynthesisGui.java

```diff
@@ -362,6 +370,12 @@ public void run() {
 
         volume.setValue((int) MathUtils.round(speech.getVolume() * 100, 0));
         lastUtterance.setText(speech.getlastUtterance());
+        if (speech.isReady()) {
+          isReadyIcon.setIcon(readyOK);
+        } else {
+          isReadyIcon.setIcon(readyNOK);
+        }
+
         restoreListeners();
       }
     });

```

Comment:
cool .. nice enhancement - and in the correct place ;)  
One Abstract to rule them All !

Reply:
the initial idea was to use a general abstract, for example a little pic in the tab, but that will do for the moment ...